static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -76,6 +79,11 @@ export function getCoverageColorClass(
       case Coverage.NOT_APPLICABLE:
       // fallthrough
       default:
+        if (lineNo === activeLineNo) {
```

Comment:
Would we want to set `primaryLineCovered = False` in `case Coverage.NOT_COVERED`?

Reply:
I thought `case Coverage.NOT_COVERED` was the case where codecov got the coverage info and it was red

Reply:
Ah yup you're right - got tripped up by the wording!